corehq/apps/reports/standard/monitoring.py

```diff
@@ -624,7 +733,7 @@ def total_active_count(self):
                 return 0
 
         def header(self):
-            return self.report.get_user_link(self.user)['html']
+            return self.report.get_user_link(self.row_data)['html']
```

Comment:
Hey @zandre-eng 

Can you drop a note here why we changed this?

I am working on a [bugfix](https://dimagi.sentry.io/issues/5330203551/?environment=production&project=136860&query=is%3Aunresolved+domain%3Agivedirectly-kenya&referrer=issue-stream&statsPeriod=24h&stream_index=1) and wondering that this passes row_data instead of user which I assume would be a User object & is not expected by the function `get_user_link`.

Reply:
okay, we changed the name of the variable from user to row_data. Can you share why that was changed?

Reply:
This variable was changed to represent that a User object should no longer be passed here, but rather an instance of `RowData`. This change was done because `Row` now supports both users and locations/groups. To make it reusable for both it's keeping more generic row data instead of data for a specific user.

Looking at the bug, it seems the error is happening because the conditional [here](https://github.com/dimagi/commcare-hq/blob/300531bdbfbfea9240795a6d1f737b6b23cd3093/corehq/apps/reports/standard/monitoring.py#L99) is not passing, hence causing `name` to never be initialized. The fix for this seems to be to simply initialize the `name` var outside of the conditional.

Reply:
Cool.

Seems reasonable to change what was being passed, though we didn't change the name of the parameter in the function `get_user_link` and that seems to be causing some confusion.
For [example](https://github.com/dimagi/commcare-hq/blob/300531bdbfbfea9240795a6d1f737b6b23cd3093/corehq/apps/reports/standard/monitoring.py#L101C13-L101C85):
`name = user.raw_username if hasattr(user, 'raw_username') else user.name`

Reply:
Ah, that makes sense. I think this function could be renamed to `get_row_link` or something of the sort. Looking at its references, it's only being used in `monitoring.py` so should be safe to change the function name.

Reply:
and the parameter as well. I was looking at its usages and I see user is passed at some, so I am a bit confused that this might need more looking into, unless I looked at usages for a different function with the same name